src/storage/src/hummock/iterator/backward_user.rs

```diff
@@ -53,6 +53,8 @@ pub struct BackwardUserIterator {
     /// Only reads values if `epoch <= self.read_epoch`.
     read_epoch: Epoch,
 
+    min_epoch: Epoch,
```

Comment:
Please add some comments for the usage of this field. Should explicitly mentioned that this is only be used for ttl.